app/bundles/EmailBundle/Swiftmailer/Momentum/Service/SwiftMessageService.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace Mautic\EmailBundle\Swiftmailer\Momentum\Service;
+
+use Mautic\EmailBundle\Helper\PlainTextMassageHelper;
+use Mautic\EmailBundle\Swiftmailer\Message\MauticMessage;
+use Mautic\EmailBundle\Swiftmailer\Momentum\DTO\TransmissionDTO;
+use Mautic\EmailBundle\Swiftmailer\Momentum\Metadata\MetadataProcessor;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Class SwiftMessageService.
+ */
+final class SwiftMessageService implements SwiftMessageServiceInterface
```

Comment:
This class is again too large, testing will be very difficult. I suggest to split it in a similar way how I did it with SendGrid - Separate class for attachments, recipients and so on.